src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs

```diff
@@ -135,7 +143,8 @@ private void UpdateTree()
 
             if (viewModelCount > 0)
             {
-                listControl.lvDetails.SelectedItem = 0;
+                nonFrameworkListControl.lvDetails.SelectedItem = 0;
```

Comment:
We should probably only set 1 of these -- nonFrameworkListControl if we have any non-framework issues, otherwise frameworkListControl